app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
+    app.register_blueprint(descriptor_blueprint, url_prefix='/descriptor')
+
+    from resource import resource as resource_blueprint
```

Comment:
I mean not right now, but possibly later.
Also bulk_crud doesn't make much sense, since U won't be done in bulk, and R is always in bulk.